app/views/devise/registrations/edit.html.erb

```diff
@@ -4,6 +4,21 @@
   <%= devise_error_messages! %>
 
   <div class="field">
+    <%= f.label :first_name %><br />
+    <%= f.text_field :first_name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :last_name %><br />
+    <%= f.text_field :last_name, autofocus: true %>
+  </div>
+
+  <div class="field">
+    <%= f.label :username %><br />
+    <%= f.text_field :username, autofocus: true %>
```

Comment:
You shouldn't have `autofocus: true` on all of these - that attribute refers to which form tag should start with a cursor in it when the page loads, which shouldn't (and can't) be all of them! Just pick one - preferably the first one